From 8be27b6d10527f7e0a30cf0ad13f3241acb87c46 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@nowhere.com>
Date: Mon, 21 Dec 2015 01:41:00 +0800
Subject: [PATCH] fix angle configuration file

---
 libEGL.pc.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/libEGL.pc.in b/libEGL.pc.in
index dd4e52a..9ad5722 100644
--- a/libEGL.pc.in
+++ b/libEGL.pc.in
@@ -1,6 +1,7 @@
 prefix=%PREFIX%
-libdir=%LIBDIR%
-includedir=%INCLUDEDIR%
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
 
 Name: libEGL
 Description: A conformant OpenGL ES 2 implementation for Windows.
-- 
2.6.2

