From a80aa5691ea0f8a1b41affcb0818ba6638ee9250 Mon Sep 17 00:00:00 2001
From: myfreeer <myfreeer@users.noreply.github.com>
Date: Tue, 23 May 2017 14:37:08 +0800
Subject: [PATCH] fix d3d11 error

---
 src/libANGLE/renderer/d3d/d3d11/ResourceManager11.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libANGLE/renderer/d3d/d3d11/ResourceManager11.h b/src/libANGLE/renderer/d3d/d3d11/ResourceManager11.h
index 6daa9f04b..023ace09e 100644
--- a/src/libANGLE/renderer/d3d/d3d11/ResourceManager11.h
+++ b/src/libANGLE/renderer/d3d/d3d11/ResourceManager11.h
@@ -15,6 +15,7 @@
 #include "common/angleutils.h"
 #include "common/debug.h"
 #include "libANGLE/Error.h"
+#include "libANGLE/renderer/d3d/d3d11/renderer11_utils.h"
 
 namespace rx
 {
-- 
2.12.1

