From 9dd795ad964484405759f0594304a58b89ada09a Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Fri, 29 Jul 2016 18:37:44 +0800
Subject: [PATCH] fix include headers

---
 src/libANGLE/renderer/d3d/DisplayD3D.cpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/libANGLE/renderer/d3d/DisplayD3D.cpp b/src/libANGLE/renderer/d3d/DisplayD3D.cpp
index 38b49f9..c276afd 100644
--- a/src/libANGLE/renderer/d3d/DisplayD3D.cpp
+++ b/src/libANGLE/renderer/d3d/DisplayD3D.cpp
@@ -7,11 +7,10 @@
 // DisplayD3D.cpp: D3D implementation of egl::Display
 
 // These headers annoyingly have to be the first lines in the file.
-#include <Windows.h>
+#include <windows.h>
 #include <cfgmgr32.h>
-#include <strsafe.h>
 #include <setupapi.h>
-#include <VersionHelpers.h>
+#include <versionhelpers.h>
 
 #include "libANGLE/renderer/d3d/DisplayD3D.h"
 
-- 
2.9.1

