From af5f65d04767e3decfaeaf34139813505dbc8bce Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Tue, 23 Aug 2022 18:00:20 +0800
Subject: [PATCH] remove -static from library's name

---
 CMakeLists.txt | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e86b13b..aa0873c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -219,6 +219,13 @@ if(BROTLI_PARENT_DIRECTORY)
   set(BROTLI_LIBRARIES "${BROTLI_LIBRARIES}" PARENT_SCOPE)
 endif()
 
+# Remove -static in library's names
+if (NOT WIN32 OR MINGW)
+  set_target_properties(brotlicommon-static PROPERTIES OUTPUT_NAME "brotlicommon")
+  set_target_properties(brotlidec-static PROPERTIES OUTPUT_NAME "brotlidec")
+  set_target_properties(brotlienc-static PROPERTIES OUTPUT_NAME "brotlienc")
+endif()
+
 # Build the brotli executable
 add_executable(brotli ${BROTLI_CLI_C})
 target_link_libraries(brotli ${BROTLI_LIBRARIES_STATIC})
@@ -230,7 +237,7 @@ if(NOT BROTLI_BUNDLED_MODE)
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
   )
 
-  if(NOT BROTLI_EMSCRIPTEN)
+  if(NOT BROTLI_EMSCRIPTEN AND BUILD_SHARED_LIBS)
     install(
       TARGETS ${BROTLI_LIBRARIES_CORE}
       ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-- 
2.37.2

