From 5f2fc82bedb6f50f67e57d7ec8b3b4e486e6475a Mon Sep 17 00:00:00 2001
From: Hiltjo Posthuma <hiltjo@codemadness.org>
Date: Mon, 19 Jan 2015 22:02:35 +0000
Subject: [PATCH] mingw compatibility for mkstemp, make_hash tool crosscompile

---
 src/filebuf.c     | 13 +++++++++++++
 tools/Makefile.am |  2 +-
 2 files changed, 14 insertions(+), 1 deletion(-)

diff --git a/src/filebuf.c b/src/filebuf.c
index 0a47b79..66f8d3f 100644
--- a/src/filebuf.c
+++ b/src/filebuf.c
@@ -301,6 +301,19 @@ file_write(File *file)
   return bw;
 }
 
+#if defined(__MINGW32__)
+int
+mkstemp(char *template)
+{
+	char *filename;
+
+	filename = mktemp(template);
+	if (filename == NULL)
+		return 1;
+	return open(filename, O_RDWR | O_CREAT, 0600);
+}
+#endif
+
 /* create temporary file, open it readwrite and return the created File
    retuns NULL when we are unable to create a temporary file
    when ulink is true, delete it right after opening
diff --git a/tools/Makefile.am b/tools/Makefile.am
index b612812..bf2e7b6 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -31,7 +31,7 @@ encodings.sed: $(top_builddir)/iconvenc.h
 	sed -e 's/^#define \([A-Z0-9_]*\) \(.*\)/@\1@ \2/' -e 's/"//g' -e 's/NULL$$//' -e 's/ /\//' -e 's/^\(.*\)$$/s\/\1\//' $(top_builddir)/iconvenc.h >encodings.sed
 
 encodings.h: encodings.sed $(srcdir)/encodings.dat make_hash
-	sed -f encodings.sed $(srcdir)/encodings.dat | ./make_hash >encodings.h
+	sed -f encodings.sed $(srcdir)/encodings.dat | wine ./make_hash >encodings.h
 
 # Normally there's no need to regenerate tables, they are copy-and-pasted
 # into the C source manually, but the rules are here.  Run `make tables'
-- 
2.1.2

