From c4018b79162c38e9c35d3f7b26ee254129692110 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 7 Sep 2013 15:43:34 +0200
Subject: [PATCH] Lazy cross compile fix

---
 configure.ac      | 3 ---
 tools/Makefile.am | 2 +-
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5ab6ffb..f3b266e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -97,9 +97,6 @@ done
 dnl And for gtk-doc
 gtk_CHECK_GTK_DOC
 
-dnl Check for good random number sources
-AC_CHECK_FILES(/dev/random /dev/urandom /dev/srandom /dev/arandom)
-
 dnl Dirty path hack.  Helps some people with badly set up search paths.
 if test "$prefix" = "NONE"; then
   LDFLAGS="$LDFLAGS -L$ac_default_prefix/lib"
diff --git a/tools/Makefile.am b/tools/Makefile.am
index b612812..bf2e7b6 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -31,7 +31,7 @@ encodings.sed: $(top_builddir)/iconvenc.h
 	sed -e 's/^#define \([A-Z0-9_]*\) \(.*\)/@\1@ \2/' -e 's/"//g' -e 's/NULL$$//' -e 's/ /\//' -e 's/^\(.*\)$$/s\/\1\//' $(top_builddir)/iconvenc.h >encodings.sed
 
 encodings.h: encodings.sed $(srcdir)/encodings.dat make_hash
-	sed -f encodings.sed $(srcdir)/encodings.dat | ./make_hash >encodings.h
+	sed -f encodings.sed $(srcdir)/encodings.dat | wine ./make_hash >encodings.h
 
 # Normally there's no need to regenerate tables, they are copy-and-pasted
 # into the C source manually, but the rules are here.  Run `make tables'
-- 
1.8.3.4

