From e7d075f88428ef74baaae1720a107230db4b6e4e Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 2 Nov 2013 01:16:25 +0100
Subject: [PATCH] Fix mktemp_s

---
 configure.ac   | 2 +-
 src/fccompat.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4478914..44ab993 100644
--- a/configure.ac
+++ b/configure.ac
@@ -148,7 +148,7 @@ AC_TYPE_PID_T
 # Checks for library functions.
 AC_FUNC_VPRINTF
 AC_FUNC_MMAP
-AC_CHECK_FUNCS([link mkstemp mkostemp _mktemp_s mkdtemp getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r regcomp regerror regexec regfree fstatvfs fstatfs lstat])
+AC_CHECK_FUNCS([link mkstemp mkostemp _mktemp mkdtemp getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r regcomp regerror regexec regfree fstatvfs fstatfs lstat])
 
 dnl AC_CHECK_FUNCS doesn't check for header files.
 dnl posix_fadvise() may be not available in older libc.
diff --git a/src/fccompat.c b/src/fccompat.c
index 1c2ba8b..1a42c91 100644
--- a/src/fccompat.c
+++ b/src/fccompat.c
@@ -152,8 +152,8 @@ FcMakeTempfile (char *template)
 	fcntl(fd, F_SETFD, fcntl(fd, F_GETFD) | FD_CLOEXEC);
     }
 #  endif
-#elif HAVE__MKTEMP_S
-   if (_mktemp_s(template, strlen(template) + 1) != 0)
+#elif HAVE__MKTEMP
+   if (_mktemp(template) != 0)
        return -1;
    fd = FcOpen(template, O_RDWR | O_EXCL | O_CREAT, 0600);
 #endif
-- 
1.8.0.1

