From cb1e9b3a1d6ea44af4a7e1761d72a39a3d6cbef7 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Fri, 27 Feb 2015 16:55:43 +0100
Subject: [PATCH] =?UTF-8?q?Don=E2=80=99t=20build=20unit=20tests?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 2b4a5b8..26f46c0 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -23,7 +23,7 @@

 SUBDIRS=fontconfig fc-blanks fc-case fc-lang fc-glyphname src \
 	fc-cache fc-cat fc-list fc-match fc-pattern fc-query fc-scan \
-	fc-validate conf.d test
+	fc-validate conf.d
 if ENABLE_DOCS
 SUBDIRS += doc
 endif
--
2.2.2

