From e6db8e7a303bafdbba59021019a614adeae5c669 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 16 Mar 2019 19:35:28 +0800
Subject: [PATCH] remove test

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6d4cd32..cdd06b7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,7 +24,7 @@
 SUBDIRS=fontconfig fc-case fc-lang src \
 	fc-cache fc-cat fc-conflist fc-list fc-match \
 	fc-pattern fc-query fc-scan fc-validate conf.d \
-	its po po-conf test
+	its po po-conf
 if ENABLE_DOCS
 SUBDIRS += doc
 endif
-- 
2.37.2

