From a8bc2a52b2b74d5c6d1fbfc9228c671a751381f9 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 7 Sep 2013 14:38:49 +0200
Subject: [PATCH 2/2] Fix mktemp_s

---
 configure.ac   | 2 +-
 src/fccompat.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 952e737..5cb96d7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -148,7 +148,7 @@ AC_TYPE_PID_T
 # Checks for library functions.
 AC_FUNC_VPRINTF
 AC_FUNC_MMAP
-AC_CHECK_FUNCS([link mkstemp mkostemp _mktemp_s getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r regcomp regerror regexec regfree fstatvfs fstatfs lstat])
+AC_CHECK_FUNCS([link mkstemp mkostemp _mktemp getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r regcomp regerror regexec regfree fstatvfs fstatfs lstat])
 
 dnl AC_CHECK_FUNCS doesn't check for header files.
 dnl posix_fadvise() may be not available in older libc.
diff --git a/src/fccompat.c b/src/fccompat.c
index d4f88c8..8761b8b 100644
--- a/src/fccompat.c
+++ b/src/fccompat.c
@@ -156,8 +156,8 @@ FcMakeTempfile (char *template)
 	fcntl(fd, F_SETFD, fcntl(fd, F_GETFD) | FD_CLOEXEC);
     }
 #  endif
-#elif HAVE__MKTEMP_S
-   if (_mktemp_s(template, strlen(template) + 1) != 0)
+#elif HAVE__MKTEMP
+   if (_mktemp(template) != 0)
        return -1;
    fd = FcOpen(template, O_RDWR | O_EXCL | O_CREAT, 0600);
 #endif
-- 
1.8.3.4

