From 2af955e2941ca0ce31475f8e41dfbb4cfbe1229e Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 16 Mar 2019 19:36:16 +0800
Subject: [PATCH 2/2] fix utimes

---
 src/fccache.c | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/src/fccache.c b/src/fccache.c
index ac46ee8..b98ed69 100644
--- a/src/fccache.c
+++ b/src/fccache.c
@@ -45,7 +45,16 @@
 #ifndef O_BINARY
 #define O_BINARY 0
 #endif
-
+#ifdef __MINGW32__
+#include <sys/utime.h>
+int utimes(const char *filename, const struct timeval times[2]) {
+	struct _utimbuf timebuffer;
+	timebuffer.actime = times[0].tv_sec;
+	timebuffer.modtime = times[1].tv_sec;
+
+	return _utime(filename, &timebuffer);
+}
+#endif
 FcBool
 FcDirCacheCreateUUID (FcChar8  *dir,
 		      FcBool    force,
@@ -55,6 +64,7 @@ FcDirCacheCreateUUID (FcChar8  *dir,
     FcChar8 *target;
     FcBool ret = FcTrue;
 #ifndef _WIN32
+
     FcChar8 *uuidname;
 
     if (sysroot)
-- 
2.17.1

