From cdb4398b525e171631af9163016c994354925fe2 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 15:14:57 +0100
Subject: [PATCH 7/7] Always use -lintl

---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 713a3f0..5096995 100644
--- a/configure.ac
+++ b/configure.ac
@@ -459,6 +459,7 @@ if test "$gt_cv_have_gettext" != "yes" ; then
 ])
 fi
 
+INTLLIBS="-lintl $INTLLIBS"
 LIBS="$INTLLIBS $LIBS"
 
 GETTEXT_PACKAGE=glib20
-- 
1.8.5

