From 664934c8ccf56f20a52b336c9f2e7503f9e12c24 Mon Sep 17 00:00:00 2001
From: MXE <mxe@mxe.cc>
Date: Fri, 15 Jun 2012 15:25:01 +0200
Subject: [PATCH 3/7] Avoid DllMain symbol conflict when linking statically

---
 gio/giomodule.c  | 8 ++------
 glib/glib-init.c | 4 ++--
 2 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/gio/giomodule.c b/gio/giomodule.c
index 043ec19..732053f 100644
--- a/gio/giomodule.c
+++ b/gio/giomodule.c
@@ -904,14 +904,12 @@ extern GType g_gtk_notification_backend_get_type (void);
 
 static HMODULE gio_dll = NULL;
 
-#ifdef DLL_EXPORT
-
-BOOL WINAPI DllMain (HINSTANCE hinstDLL,
+BOOL WINAPI gio_DllMain (HINSTANCE hinstDLL,
                      DWORD     fdwReason,
                      LPVOID    lpvReserved);
 
 BOOL WINAPI
-DllMain (HINSTANCE hinstDLL,
+gio_DllMain (HINSTANCE hinstDLL,
 	 DWORD     fdwReason,
 	 LPVOID    lpvReserved)
 {
@@ -921,8 +919,6 @@ DllMain (HINSTANCE hinstDLL,
   return TRUE;
 }
 
-#endif
-
 void *
 _g_io_win32_get_module (void)
 {
diff --git a/glib/glib-init.c b/glib/glib-init.c
index df812c7..5dec781 100644
--- a/glib/glib-init.c
+++ b/glib/glib-init.c
@@ -234,14 +234,14 @@ glib_init (void)
 
 #if defined (G_OS_WIN32)
 
-BOOL WINAPI DllMain (HINSTANCE hinstDLL,
+BOOL WINAPI glib_DllMain (HINSTANCE hinstDLL,
                      DWORD     fdwReason,
                      LPVOID    lpvReserved);
 
 HMODULE glib_dll;
 
 BOOL WINAPI
-DllMain (HINSTANCE hinstDLL,
+glib_DllMain (HINSTANCE hinstDLL,
          DWORD     fdwReason,
          LPVOID    lpvReserved)
 {
-- 
1.8.5

