From f842667c19ff9ac6bfc0cc7e2c5370c3dafa7892 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 14 Jun 2014 17:44:05 +0200
Subject: [PATCH 8/8] gobject/gtype: include stdint.h

---
 gobject/gtype.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gobject/gtype.c b/gobject/gtype.c
index e33a612..cb1adb3 100644
--- a/gobject/gtype.c
+++ b/gobject/gtype.c
@@ -21,6 +21,7 @@
 
 #include "config.h"
 
+#include <stdint.h>
 #include "../glib/valgrind.h"
 #include <string.h>
 
-- 
1.9.3

