From 3bfb2ef16ca58d361fbc97a900b008e7d1fda4b4 Mon Sep 17 00:00:00 2001
From: MXE <mxe@mxe.cc>
Date: Fri, 15 Jun 2012 15:29:06 +0200
Subject: [PATCH 5/8] Link with dnsapi

---
 gio-2.0.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio-2.0.pc.in b/gio-2.0.pc.in
index a61eaaf..11fb6f4 100644
--- a/gio-2.0.pc.in
+++ b/gio-2.0.pc.in
@@ -13,6 +13,6 @@ Description: glib I/O library
 Version: @VERSION@
 Requires: glib-2.0 gobject-2.0
 Requires.private: gmodule-no-export-2.0
-Libs: -L${libdir} -lgio-2.0
+Libs: -L${libdir} -lgio-2.0 -ldnsapi -liphlpapi
 Libs.private: @ZLIB_LIBS@ @NETWORK_LIBS@ @SELINUX_LIBS@
 Cflags:
-- 
1.8.5

