From dce3ad5987efa9b41741bde7f8ab159618d52eff Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 23 Feb 2020 19:04:18 +0800
Subject: [PATCH] Build library only

---
 configure.ac    | 8 +-------
 src/Makefile.am | 5 -----
 2 files changed, 1 insertion(+), 12 deletions(-)

diff --git a/configure.ac b/configure.ac
index c3d1c59..126bf65 100644
--- a/configure.ac
+++ b/configure.ac
@@ -14,11 +14,6 @@ AC_PROG_CC
 AC_PROG_LIBTOOL
 PKG_PROG_PKG_CONFIG
 
-# Checks for libraries.
-PKG_CHECK_EXISTS([sndfile], [], [
-    AC_MSG_ERROR(Please install libsndfile.)
-])
-
 # Checks for header files.
 AC_CHECK_HEADERS([fcntl.h malloc.h string.h])
 
@@ -27,8 +22,7 @@ AC_HEADER_STDBOOL
 AC_TYPE_SIZE_T
 
 # Checks for library functions.
-AC_FUNC_MALLOC
-AC_CHECK_FUNCS([strrchr])
+AC_CHECK_FUNCS([malloc strrchr])
 
 AC_CONFIG_FILES([libbs2b.pc
                  Makefile
diff --git a/src/Makefile.am b/src/Makefile.am
index 414aeec..98ed494 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,11 +1,6 @@
 lib_LTLIBRARIES = \
 	libbs2b.la
 
-bin_PROGRAMS = \
-	bs2bconvert \
-	bs2bstream
-
-
 bs2b_HEADERS = \
 	bs2b.h \
 	bs2bclass.h \
-- 
2.17.1

