--- potomo
+++ potomo
@@ -57,7 +57,7 @@ case "$fromset" in
         ;;   
     *)
         echo "potomo: '$infile' converting from $fromset to utf-8" >&2
-        iconv --silent --from-code=$fromset --to-code=utf-8 < "$infile" |\
+        iconv -s --from-code=$fromset --to-code=utf-8 < "$infile" |\
             sed "/^\"Content-Type:/ s/charset=[a-zA-Z0-9_-]*/charset=utf-8/"|\
             msgfmt --output-file="$outfile" -
         ;;
