From 6806b3754515e5f5e296d70fe6e2f7b9bd87e106 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Thu, 30 Jul 2015 21:46:06 +0200
Subject: [PATCH] Remove typedef clashes with Windows headers

---
 jmorecfg.h | 18 +-----------------
 1 file changed, 1 insertion(+), 17 deletions(-)

diff --git a/jmorecfg.h b/jmorecfg.h
index 108e7de..07c3123 100644
--- a/jmorecfg.h
+++ b/jmorecfg.h
@@ -21,7 +21,7 @@
  * really short on memory.  (Each allowed component costs a hundred or so
  * bytes of storage, whether actually used in an image or not.)
  */
-
+#include <windows.h>
 #define MAX_COMPONENTS  10      /* maximum number of image components */
 
 
@@ -145,12 +145,6 @@ typedef unsigned int UINT16;
 typedef short INT16;
 #endif
 
-/* INT32 must hold at least signed 32-bit values. */
-
-#ifndef XMD_H                   /* X11/xmd.h correctly defines INT32 */
-typedef long INT32;
-#endif
-
 /* Datatype used for image dimensions.  The JPEG standard only supports
  * images up to 64K*64K due to 16-bit fields in SOF markers.  Therefore
  * "unsigned int" is sufficient on all machines.  However, if you need to
@@ -198,16 +192,6 @@ typedef unsigned int JDIMENSION;
 #define FAR
 
 
-/*
- * On a few systems, type boolean and/or its values FALSE, TRUE may appear
- * in standard header files.  Or you may have conflicts with application-
- * specific header files that you want to include together with these files.
- * Defining HAVE_BOOLEAN before including jpeglib.h should make it work.
- */
-
-#ifndef HAVE_BOOLEAN
-typedef int boolean;
-#endif
 #ifndef FALSE                   /* in case these macros already exist */
 #define FALSE   0               /* values of boolean */
 #endif
-- 
2.4.5

