From 630c5cc765ba01836c883450c70185d3b9d15c23 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 14:01:14 +0100
Subject: [PATCH 5/6] Remove retarded check for header presence

---
 src/quvi-0.9/quvi/Makefile.am | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/quvi-0.9/quvi/Makefile.am b/src/quvi-0.9/quvi/Makefile.am
index e298133..e9a0869 100644
--- a/src/quvi-0.9/quvi/Makefile.am
+++ b/src/quvi-0.9/quvi/Makefile.am
@@ -16,12 +16,4 @@ libquvisubinclude_HEADERS=\
   qsupp.h\
   qversion.h
 
-install-data-local:
-	@if test -f $(libquvisubincludedir)/qfunc.h ; then \
-		echo "** Old headers found in $(libquvisubincludedir).  You should" ; \
-		echo "** remove the contents of this directory and type 'make " ; \
-		echo "** install' again." ; \
-		false ; \
-	fi
-
 # vim: set ts=2 sw=2 tw=72 expandtab:
-- 
1.8.5

