From 5c0f97a08e0159ab2cbc1e721ce5963bc0a3b2b6 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 12:44:30 +0100
Subject: [PATCH 2/6] configure: remove check for scriptsdir existence

---
 configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index bb82d53..337789a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -114,9 +114,6 @@ AC_ARG_WITH([scriptsdir],
 AS_IF([test -z "$scriptsdir"], [
   AC_MSG_ERROR([scriptsdir: variable not found])])
 
-AS_IF([ ! test -d "$scriptsdir"], [
-  AC_MSG_ERROR([$scriptsdir: does not exist or is not a directory])])
-
 AS_IF([test -z "$versionfile"], [
   AC_MSG_ERROR([versionfile: variable not found])])
 
-- 
1.8.5

