From 762647e642c4363ce4a595f92f680417fed91342 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 13:29:27 +0100
Subject: [PATCH 4/6] Do not build examples and tests

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index c05fa39..239c3a2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,7 @@ EXTRA_DIST=libquvi-0.9.pc.in gen-ver.sh VERSION
 pkgconfigdir=	$(libdir)/pkgconfig
 pkgconfig_DATA=	libquvi-0.9.pc
 
-SUBDIRS=po src examples doc tests
+SUBDIRS=po src doc
 
 .PHONY: doc distdoc distdoc-html distdoc-latex distdoc-pdf ChangeLog VERSION test test-mem
 
-- 
1.8.5

