From f3d393b048fbbbcd95b467f66437068891153459 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 18:04:58 +0100
Subject: [PATCH] Do not install symlinks

---
 share/Makefile.am | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/share/Makefile.am b/share/Makefile.am
index 2d32ba2..c3d0d78 100644
--- a/share/Makefile.am
+++ b/share/Makefile.am
@@ -127,14 +127,13 @@ dist_scan_DATA=\
   scan/youtube.lua
 
 install-data-hook:
-	cd $(DESTDIR)$(pkgdatadir) && rm -f $(VERSION_MM) && \
-		$(LN_S) $(VERSION) $(VERSION_MM) ; \
+	cd $(DESTDIR)$(pkgdatadir) && rm -rf $(VERSION_MM) && \
+		mv $(VERSION) $(VERSION_MM) ; \
 	cd $(VERSION_MM) && echo "[libquvi-scripts]" >version && \
 		echo "configuration = $(BUILD_OPTS)" >>version && \
 		echo "version = v$(VERSION)" >>version
 
 uninstall-hook:
-	rm -rf "$(DESTDIR)$(pkgdatadir)/$(VERSION)" ; \
-	rm -f "$(DESTDIR)$(pkgdatadir)/$(VERSION_MM)"
+	rm -rf "$(DESTDIR)$(pkgdatadir)/$(VERSION_MM)"
 
 # vim: set ts=2 sw=2 tw=72 expandtab:
-- 
1.8.5

