From d3d6e7e95087613effbc67926a75b90a503837f5 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 15:05:44 +0100
Subject: [PATCH 6/6] Add static linking dependencies to pkg-config file

---
 libquvi-0.9.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libquvi-0.9.pc.in b/libquvi-0.9.pc.in
index 280525d..dbab0db 100644
--- a/libquvi-0.9.pc.in
+++ b/libquvi-0.9.pc.in
@@ -8,4 +8,5 @@ URL: http://quvi.sourceforge.net/
 Description: Query media library for Youtube like websites
 Version: @VN_C@
 Libs: -L${libdir} -lquvi-0.9
+Libs.private: @liblua_LIBS@ @libcurl_LIBS@ @glib_LIBS@ @LIBGCRYPT_LIBS@
 Cflags: -I${includedir}/quvi-0.9
-- 
1.8.5

