From f8b87340c6d30bdc458027c955eacfd85e8cc585 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Fri, 6 May 2022 16:43:43 +0800
Subject: [PATCH] upgrade to mbedtls v3

---
 src/pki_mbedcrypto.c | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

diff --git a/src/pki_mbedcrypto.c b/src/pki_mbedcrypto.c
index 52b298e..2b64010 100644
--- a/src/pki_mbedcrypto.c
+++ b/src/pki_mbedcrypto.c
@@ -119,18 +119,20 @@ ssh_key pki_private_key_from_base64(const char *b64_key, const char *passphrase,
                     valid = mbedtls_pk_parse_key(rsa,
                             (const unsigned char *) b64_key,
                             b64len, tmp,
-                            strnlen((const char *) tmp, MAX_PASSPHRASE_SIZE));
+                            strnlen((const char *) tmp, MAX_PASSPHRASE_SIZE),
+                            mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
                 } else {
                     valid = mbedtls_pk_parse_key(rsa,
                             (const unsigned char *) b64_key,
                             b64len, NULL,
-                            0);
+                            0, mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
                 }
             } else {
                 valid = mbedtls_pk_parse_key(rsa,
                         (const unsigned char *) b64_key, b64len,
                         (const unsigned char *) passphrase,
-                        strnlen(passphrase, MAX_PASSPHRASE_SIZE));
+                        strnlen(passphrase, MAX_PASSPHRASE_SIZE),
+                        mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
             }
 
             if (valid != 0) {
@@ -160,18 +162,20 @@ ssh_key pki_private_key_from_base64(const char *b64_key, const char *passphrase,
                     valid = mbedtls_pk_parse_key(ecdsa,
                             (const unsigned char *) b64_key,
                             b64len, tmp,
-                            strnlen((const char *) tmp, MAX_PASSPHRASE_SIZE));
+                            strnlen((const char *) tmp, MAX_PASSPHRASE_SIZE),
+                            mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
                 } else {
                     valid = mbedtls_pk_parse_key(ecdsa,
                             (const unsigned char *) b64_key,
                             b64len, NULL,
-                            0);
+                            0, mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
                 }
             } else {
                 valid = mbedtls_pk_parse_key(ecdsa,
                         (const unsigned char *) b64_key, b64len,
                         (const unsigned char *) passphrase,
-                        strnlen(passphrase, MAX_PASSPHRASE_SIZE));
+                        strnlen(passphrase, MAX_PASSPHRASE_SIZE),
+                        mbedtls_ctr_drbg_random, ssh_get_mbedtls_ctr_drbg_context());
             }
 
             if (valid != 0) {
@@ -1079,6 +1083,7 @@ static ssh_string rsa_do_sign_hash(const unsigned char *digest,
                          digest,
                          dlen,
                          sig,
+                         sizeof sig,
                          &slen,
                          mbedtls_ctr_drbg_random,
                          ssh_get_mbedtls_ctr_drbg_context());
-- 
2.36.0

