From e3786bef288580efd5688c869f2c3ae9b25b621e Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 3 Oct 2021 11:36:34 +0800
Subject: [PATCH] Install modified .pc file

---
 CMakeLists.txt  |  2 +-
 libssh.pc.cmake | 16 ++++++++++++----
 2 files changed, 13 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 32172d9..a295551 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,7 +125,7 @@ add_subdirectory(src)
 
 # pkg-config file
 if (UNIX OR MINGW)
-configure_file(libssh.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libssh.pc)
+configure_file(libssh.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libssh.pc @ONLY)
 install(
   FILES
     ${CMAKE_CURRENT_BINARY_DIR}/libssh.pc
diff --git a/libssh.pc.cmake b/libssh.pc.cmake
index b37cb3f..2b2f25c 100644
--- a/libssh.pc.cmake
+++ b/libssh.pc.cmake
@@ -1,6 +1,14 @@
-Name: ${PROJECT_NAME}
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include/libssh
+
+Name: @PROJECT_NAME@
 Description: The SSH Library
-Version: ${PROJECT_VERSION}
-Libs: -L${CMAKE_INSTALL_FULL_LIBDIR} -lssh
-Cflags: -I${CMAKE_INSTALL_FULL_INCLUDEDIR}
+Version: @PROJECT_VERSION@
+
+Requires.private: libssl zlib
+Libs: -L${libdir} -lssh
+Libs.private: -L${libdir} -lssl -lz
+Cflags: -I${includedir} -DLIBSSH_STATIC
 
-- 
2.33.0

