From dee9bb9aadc86b91e5daf6012e4efef3e7d39360 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Thu, 21 Apr 2022 15:49:04 +0800
Subject: [PATCH] avoid name collision

---
 srtcore/md5.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/srtcore/md5.h b/srtcore/md5.h
index 98bd076..75e4c8a 100644
--- a/srtcore/md5.h
+++ b/srtcore/md5.h
@@ -50,6 +50,14 @@
 #ifndef md5_INCLUDED
 #  define md5_INCLUDED
 
+/*
+ * To avoid the collision with libssh, lets
+ * rename the methods here with some defines
+ */
+#define md5_init srt_md5_init
+#define md5_append srt_md5_append
+#define md5_finish srt_md5_finish
+
 /*
  * All symbols have been put under the srt namespace
  * to avoid potential linkage conflicts.
-- 
2.35.2

