From febc1d713c91a51d79483ebabf77df63ef0cc86e Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 24 Sep 2022 07:00:18 +0800
Subject: [PATCH] Install modified .pc file

---
 libssh.pc.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libssh.pc.cmake b/libssh.pc.cmake
index f288b94d..1673fb7a 100644
--- a/libssh.pc.cmake
+++ b/libssh.pc.cmake
@@ -7,4 +7,7 @@ Name: @PROJECT_NAME@
 Description: The SSH Library
 Version: @PROJECT_VERSION@
 Libs: -L${libdir} -lssh
+Libs.private: -lwsock32 -lpthread
+Requires.private: libssl
 Cflags: -I${includedir}
+Cflags.private: -DLIBSSH_STATIC
-- 
2.42.1

