From dd8f382407547d84da0e6e2054ad4855ec083d3c Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 19 Mar 2023 14:45:02 +0800
Subject: [PATCH] skip directory

---
 Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 38e8d06..7a68b2d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -21,9 +21,6 @@ SUBDIRS = \
 	m4 \
 	src \
 	$(proxy_dirs) \
-	contrib \
-	examples \
-	test \
 	po \
 	doc
 
-- 
2.39.2

