From 85f010d70335fe7351a58ff4b1f2275d935853b4 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 19 Mar 2023 14:23:05 +0800
Subject: [PATCH 3/5] fix static linking

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 915875a..7daa4c9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -330,7 +330,7 @@ dnl (PNG page export)
 dnl
 HAVE_PNG="yes"
 AC_CHECK_LIB(png, png_destroy_write_struct,
-  LIBS="$LIBS -lpng -lz", HAVE_PNG="no", -lz -lm)
+  LIBS="-lpng -lz $LIBS", HAVE_PNG="no", -lz -lm)
 if test "x$HAVE_PNG" = xyes; then
   AC_DEFINE(HAVE_LIBPNG, 1, [Define if you have libpng])
 fi
-- 
2.39.2

