From 5ec12c09a6823beb286fef2a1367445128af44cc Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 19 Mar 2023 14:19:32 +0800
Subject: [PATCH 2/5] ioctl

---
 contrib/ntsc-cc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/ntsc-cc.c b/contrib/ntsc-cc.c
index 5f20a8d..cbeccb1 100644
--- a/contrib/ntsc-cc.c
+++ b/contrib/ntsc-cc.c
@@ -32,7 +32,6 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <locale.h>
-#include <sys/ioctl.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/time.h>
@@ -45,6 +44,7 @@
 #include "src/libzvbi.h"
 
 #ifdef ENABLE_V4L2
+#  include <sys/ioctl.h>
 #  include <asm/types.h>
 #  include "src/videodev2k.h"
 #endif
-- 
2.39.2

