From 3b68dc2dc790abc2944006088b984cba995f68d7 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Wed, 15 Jan 2014 17:30:52 +0100
Subject: [PATCH 1/2] Add LIBS to compile flags

---
 makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makefile b/makefile
index 2e419ff..e84dc5a 100644
--- a/makefile
+++ b/makefile
@@ -12,7 +12,7 @@ OBJS= src/lxplib.o $(COMPAT_O)
 lib: src/$(LIBNAME)
 
 src/$(LIBNAME) : $(OBJS)
-	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) -lexpat
+	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) $(OBJS) -lexpat $(LIBS)
 
 $(COMPAT_DIR)/compat-5.1.o: $(COMPAT_DIR)/compat-5.1.c
 	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
-- 
1.8.5

