From de9e9dde3de80a4dce44a2bc3f662f4980311ac1 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Fri, 6 Nov 2020 01:30:59 +0800
Subject: [PATCH] fix compile

---
 src/win32/fs.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/win32/fs.cpp b/src/win32/fs.cpp
index d6abdcfd3..c70e9fa91 100644
--- a/src/win32/fs.cpp
+++ b/src/win32/fs.cpp
@@ -1666,7 +1666,7 @@ bool WinFileSystemAccess::getlocalfstype(const LocalPath& path, FileSystemType&
     wstring mountPoint(MAX_PATH + 1, L'\0');
 
     if (!GetVolumePathNameW(reinterpret_cast<LPCWSTR>(tempStr.c_str()),
-                            mountPoint.data(),
+                            (LPWSTR)mountPoint.data(),
                             MAX_PATH + 1))
     {
         return type = FS_UNKNOWN, false;
@@ -1683,7 +1683,7 @@ bool WinFileSystemAccess::getlocalfstype(const LocalPath& path, FileSystemType&
                               nullptr,
                               nullptr,
                               &volumeFlags,
-                              filesystemName.data(),
+                              (LPWSTR)filesystemName.data(),
                               MAX_PATH + 1))
     {
         // Assume we can't find a matching filesystem.
-- 
2.27.0

