From 2e1d7931a4075b79269ebf00aec28235cbc5881b Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 1 Apr 2017 08:07:30 +0800
Subject: [PATCH] osc: use old default

---
 player/lua/osc.lua | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/player/lua/osc.lua b/player/lua/osc.lua
index ff2d16689..003dfbbfc 100644
--- a/player/lua/osc.lua
+++ b/player/lua/osc.lua
@@ -31,8 +31,8 @@ local user_opts = {
     iamaprogrammer = false,     -- use native mpv values and disable OSC
                                 -- internal track list management (and some
                                 -- functions that depend on it)
-    layout = "bottombar",
-    seekbarstyle = "bar",       -- slider (diamond marker), knob (circle
+    layout = "box",
+    seekbarstyle = "slider",    -- slider (diamond marker), knob (circle
                                 -- marker with guide), or bar (fill)
     title = "${media-title}",   -- string compatible with property-expansion
                                 -- to be shown as OSC title
-- 
2.12.0

