From b2f16feb3a3a95da14b6abc54c14409e4441abc0 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 13 Nov 2016 07:23:13 +0800
Subject: [PATCH] osc: use old default

---
 player/lua/osc.lua | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/player/lua/osc.lua b/player/lua/osc.lua
index 643f32f..0a0375e 100644
--- a/player/lua/osc.lua
+++ b/player/lua/osc.lua
@@ -31,8 +31,8 @@ local user_opts = {
     iamaprogrammer = false,     -- use native mpv values and disable OSC
                                 -- internal track list management (and some
                                 -- functions that depend on it)
-    layout = "bottombar",
-    seekbarstyle = "bar",       -- slider (diamond marker), knob (circle
+    layout = "box",
+    seekbarstyle = "slider",    -- slider (diamond marker), knob (circle
                                 -- marker with guide), or bar (fill)
     tooltipborder = 1,          -- border of tooltip in bottom/topbar
     timetotal = false,          -- display total time instead of remaining time?
-- 
2.9.2

