From 4ac76ada418d7b259325cf7ad2c9478f6ca6c619 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@nowhere.com>
Date: Mon, 21 Dec 2015 03:02:16 +0800
Subject: [PATCH] waf: Use pkgconfig with ANGLE

---
 wscript | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/wscript b/wscript
index 00a6700..c2f49c2 100644
--- a/wscript
+++ b/wscript
@@ -669,9 +669,7 @@ video_output_features = [
         'desc': 'OpenGL Win32 ANGLE Backend',
         'deps_any': [ 'os-win32', 'os-cygwin' ],
         'groups': [ 'gl' ],
-        'func': check_statement(['EGL/egl.h'],
-                                'eglCreateWindowSurface(0, 0, 0, 0)',
-                                lib='EGL')
+        'func': check_pkg_config('libEGL'),
     } , {
         'name': '--vdpau',
         'desc': 'VDPAU acceleration',
-- 
2.6.3

