From 643583ce3f9fd0f1456ac9d63d3610e4035d714e Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 2 Aug 2020 07:57:24 +0800
Subject: [PATCH] skip compiling testsuite and examples

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index c73a9e0..73a804d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -19,7 +19,7 @@ OPT_NETTLE_SOURCES = @OPT_NETTLE_SOURCES@
 
 FAT_TEST_LIST = @FAT_TEST_LIST@
 
-SUBDIRS = tools testsuite examples
+SUBDIRS = tools
 
 include config.make
 
-- 
2.27.0

