From 54701b0f1455838922f9c98a1344ecda1c778f44 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Mon, 4 Dec 2017 20:02:55 +0800
Subject: [PATCH] skip compiling testsuite and examples

---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 6a0c13e..b0ce215 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -17,7 +17,7 @@ OPT_HOGWEED_OBJS = @OPT_HOGWEED_OBJS@
 
 OPT_NETTLE_SOURCES = @OPT_NETTLE_SOURCES@
 
-SUBDIRS = tools testsuite examples
+SUBDIRS = tools
 
 include config.make
 
-- 
2.14.2

