From b1de5d2faaed58fb97a451bd53b9d75941a6116b Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 19 Mar 2017 12:02:42 +0000
Subject: [PATCH] Add Libs.private to openal.pc

---
 CMakeLists.txt | 5 +++++
 openal.pc.in   | 1 +
 2 files changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bff7618..df3f0a7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1202,6 +1202,11 @@ ENDIF()
 IF(LIBTYPE STREQUAL "STATIC")
     ADD_DEFINITIONS(-DAL_LIBTYPE_STATIC)
     SET(PKG_CONFIG_CFLAGS -DAL_LIBTYPE_STATIC ${PKG_CONFIG_CFLAGS})
+    FOREACH(l ${EXTRA_LIBS})
+        IF(NOT l MATCHES "^-.*")
+            SET(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${l}")
+        ENDIF()
+    ENDFOREACH()
 ENDIF()
 
 IF(NOT WIN32)
diff --git a/openal.pc.in b/openal.pc.in
index 8bdd4f3..d8759d0 100644
--- a/openal.pc.in
+++ b/openal.pc.in
@@ -8,4 +8,5 @@ Description: OpenAL is a cross-platform 3D audio API
 Requires: @PKG_CONFIG_REQUIRES@
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -l@LIBNAME@ @PKG_CONFIG_LIBS@
+Libs.private: @PKG_CONFIG_LIBS_PRIVATE@
 Cflags: -I${includedir} -I${includedir}/AL @PKG_CONFIG_CFLAGS@
-- 
2.11.1

