From 348d8f8e7c2ab6f80a57bc406224af3803decfee Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 24 Dec 2016 13:23:35 +0800
Subject: [PATCH] Add Libs.private to openal.pc

---
 CMakeLists.txt | 5 +++++
 openal.pc.in   | 1 +
 2 files changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 839bf18..08bbf4a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1172,6 +1172,11 @@ ENDIF()
 IF(LIBTYPE STREQUAL "STATIC")
     ADD_DEFINITIONS(-DAL_LIBTYPE_STATIC)
     SET(PKG_CONFIG_CFLAGS -DAL_LIBTYPE_STATIC ${PKG_CONFIG_CFLAGS})
+    FOREACH(l ${EXTRA_LIBS})
+        IF(NOT l MATCHES "^-.*")
+            SET(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${l}")
+        ENDIF()
+    ENDFOREACH()
 ENDIF()
 
 FIND_PACKAGE(Git)
diff --git a/openal.pc.in b/openal.pc.in
index 8bdd4f3..d8759d0 100644
--- a/openal.pc.in
+++ b/openal.pc.in
@@ -8,4 +8,5 @@ Description: OpenAL is a cross-platform 3D audio API
 Requires: @PKG_CONFIG_REQUIRES@
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -l@LIBNAME@ @PKG_CONFIG_LIBS@
+Libs.private: @PKG_CONFIG_LIBS_PRIVATE@
 Cflags: -I${includedir} -I${includedir}/AL @PKG_CONFIG_CFLAGS@
-- 
2.9.3

