From 6ab2cf6f67bcb09784e8db06139eda3894e9b777 Mon Sep 17 00:00:00 2001
From: Martin Herkt <lachs0r@srsfckn.biz>
Date: Sat, 7 Sep 2013 18:10:33 +0200
Subject: [PATCH] Remove useless AC checks

---
 configure.ac | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 578a11d..3a62cc4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -66,10 +66,6 @@ AC_TYPE_UINT32_T
 AC_TYPE_UINT64_T
 AC_TYPE_UINT8_T
 
-dnl Checks for library functions.
-AC_FUNC_MALLOC
-AC_CHECK_FUNCS([memset])
-
 dnl OpenCORE AMR soname version to use
 dnl goes by ‘current[:revision[:age]]’ with the soname ending up as
 dnl current.age.revision.
-- 
1.8.3.4

