From 99999e71e7f5998e6c56d432e241d5847c6aba18 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 6 Aug 2016 23:46:39 +0800
Subject: [PATCH] Append git's hash to release version

---
 update_version | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/update_version b/update_version
index a999991..9494dd9 100755
--- a/update_version
+++ b/update_version
@@ -36,7 +36,8 @@ fi
 GIT_VERSION=$(cd "$SRCDIR" && git status > /dev/null 2>&1 \
                            && git describe --tags --match 'v*' --dirty 2> /dev/null)
 GIT_VERSION=${GIT_VERSION#v}
-
+GIT_VERSION=$(git rev-parse --short @{u})
+VERSION=$(grep -o '".*"' version.mk | sed 's/"//g')
 if [ -n "$GIT_VERSION" ]; then
 
     [ "$GIT_VERSION" != "$PACKAGE_VERSION" ] || exit 1
@@ -59,7 +60,7 @@ cat > "$SRCDIR/package_version" <<-EOF
 	# to be checked and updated for every build.  Most people will
 	# never want to change this, it is an option for developers
 	# making frequent changes that they know will not be released.
-	AUTO_UPDATE=$AUTO_UPDATE
+	AUTO_UPDATE=no
 
-	PACKAGE_VERSION="$PACKAGE_VERSION"
+	PACKAGE_VERSION="$VERSION.$GIT_VERSION"
 EOF
-- 
2.9.1

