From ce143b59d274002423506234db1e326f34976e82 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sun, 9 Feb 2020 08:27:41 +0800
Subject: [PATCH] static linking hacks

---
 CMakeLists.txt | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d7e211d..c2fdadd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -332,11 +332,11 @@ if (SPIRV_CROSS_SHARED)
 	if (NOT SPIRV_CROSS_SKIP_INSTALL)
 		configure_file(
 			${CMAKE_CURRENT_SOURCE_DIR}/pkg-config/spirv-cross-c-shared.pc.in
-			${CMAKE_CURRENT_BINARY_DIR}/spirv-cross-c-shared.pc @ONLY)
-		install(FILES ${CMAKE_CURRENT_BINARY_DIR}/spirv-cross-c-shared.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+			${CMAKE_CURRENT_BINARY_DIR}/spirv-cross.pc @ONLY)
+		install(FILES ${CMAKE_CURRENT_BINARY_DIR}/spirv-cross.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 	endif()
 
-	spirv_cross_add_library(spirv-cross-c-shared spirv_cross_c_shared SHARED
+	spirv_cross_add_library(spirv-cross-c-shared spirv_cross_c_shared STATIC
 			${spirv-cross-core-sources}
 			${spirv-cross-c-sources})
 
@@ -392,11 +392,6 @@ if (SPIRV_CROSS_SHARED)
 		endif()
 	endif()
 
-	target_compile_definitions(spirv-cross-c-shared PRIVATE SPVC_EXPORT_SYMBOLS)
-
-	set_target_properties(spirv-cross-c-shared PROPERTIES
-			VERSION ${SPIRV_CROSS_VERSION}
-			SOVERSION ${spirv-cross-abi-major})
 endif()
 
 if (SPIRV_CROSS_CLI)
-- 
2.17.1

