From b4d9da104ac8d608fe16bac5a75625822226516d Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 20 Apr 2019 07:42:28 +0800
Subject: [PATCH] static linking hacks

---
 CMakeLists.txt | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 24cd320..ed0def2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -275,9 +275,9 @@ if (SPIRV_CROSS_SHARED)
 	configure_file(
 			${CMAKE_CURRENT_SOURCE_DIR}/pkg-config/spirv-cross-c-shared.pc.in
 			${CMAKE_CURRENT_BINARY_DIR}/spirv-cross-c-shared.pc @ONLY)
-	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/spirv-cross-c-shared.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pkgconfig)
+	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/spirv-cross-c-shared.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
 
-	spirv_cross_add_library(spirv-cross-c-shared spirv_cross_c_shared SHARED
+	spirv_cross_add_library(spirv-cross-c-shared spirv_cross_c_shared STATIC
 			${spirv-cross-core-sources}
 			${spirv-cross-c-sources})
 
@@ -329,12 +329,6 @@ if (SPIRV_CROSS_SHARED)
 			set_target_properties(spirv-cross-c-shared PROPERTIES LINK_FLAGS "${spirv-cross-link-flags}")
 		endif()
 	endif()
-
-	target_compile_definitions(spirv-cross-c-shared PRIVATE SPVC_EXPORT_SYMBOLS)
-
-	set_target_properties(spirv-cross-c-shared PROPERTIES
-			VERSION ${SPIRV_CROSS_VERSION}
-			SOVERSION ${spirv-cross-abi-major})
 endif()
 
 if (SPIRV_CROSS_CLI)
-- 
2.17.1

