From 265ab895329161ee7216924000d3f1dd54edb6cf Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Tue, 21 Jun 2022 15:29:10 +0800
Subject: [PATCH] remove pkg-config detection

---
 loader/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index 50cf245..2b61901 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -333,8 +333,6 @@ if (TARGET asm_offset)
 endif()
 
 # Generate pkg-config file.
-include(FindPkgConfig QUIET)
-if(PKG_CONFIG_FOUND)
     set(VK_API_VERSION "${LOADER_GENERATED_HEADER_VERSION}")
     foreach(LIB ${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES} ${PLATFORM_LIBS})
         set(PRIVATE_LIBS "${PRIVATE_LIBS} -l${LIB}")
@@ -358,7 +356,6 @@ if(PKG_CONFIG_FOUND)
     set(PRIVATE_LIBS "${PRIVATE_LIBS} -lshlwapi -lcfgmgr32")
     configure_file("vulkan.pc.in" "vulkan.pc" @ONLY)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/vulkan.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
-endif()
 
 target_link_libraries(vulkan PRIVATE Vulkan::Headers)
 add_library(Vulkan::Vulkan ALIAS vulkan)
-- 
2.37.2

