--- build/generic/configure.in_	2013-09-07 21:47:14.000000000 +0200
+++ build/generic/configure.in	2013-09-07 22:22:13.162261180 +0200
@@ -20,7 +20,7 @@
 minimum_nasm_minor_version=0
 minimum_nasm_major_version=2
 nasm_prog="nasm"
-yasm_prog="yasm"
+yasm_prog="yasm -DNO_PREFIX"
 
 dnl Default CFLAGS -- Big impact on overall speed
 our_cflags_defaults="-Wall"


--- src/encoder.h	2026-03-27 21:19:38.519038920 -0400
+++ src/encoder.h	2026-03-27 21:20:51.462035395 -0400
@@ -42,7 +42,7 @@
  * Types
  ****************************************************************************/

-typedef int bool;
+//typedef int bool;

 /*****************************************************************************
  * Structures
