From 01001c820ce946be2a5e35737165cf2ac7753055 Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@nowhere.com>
Date: Mon, 21 Dec 2015 00:10:15 +0800
Subject: [PATCH] Workaround for -isystem flag messing up include order

---
 mingw-w64-headers/crt/float.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/mingw-w64-headers/crt/float.h b/mingw-w64-headers/crt/float.h
index 5874f4e..4e56b90 100644
--- a/mingw-w64-headers/crt/float.h
+++ b/mingw-w64-headers/crt/float.h
@@ -105,6 +105,15 @@
 	#define DBL_MAX_10_EXP	__DBL_MAX_10_EXP__
 	#define LDBL_MAX_10_EXP	__LDBL_MAX_10_EXP__
 
+	/* The difference between 1 and the least value greater than 1 that is
+	representable in the given floating point type, b**1-p.  */
+	#undef FLT_EPSILON
+	#undef DBL_EPSILON
+	#undef LDBL_EPSILON
+	#define FLT_EPSILON    __FLT_EPSILON__
+	#define DBL_EPSILON    __DBL_EPSILON__
+	#define LDBL_EPSILON   __LDBL_EPSILON__
+
 	/* Addition rounds to 0: zero, 1: nearest, 2: +inf, 3: -inf, -1: unknown.  */
 	/* ??? This is supposed to change with calls to fesetround in <fenv.h>.  */
 	#undef FLT_ROUNDS
-- 
2.6.3

